modules/tfm/boards/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2021, Nordic Semiconductor ASA.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#-------------------------------------------------------------------------------
+
+# NOTE: All code in this folder and subfolders is run as part of the TF-M
+#       build, not in a Zephyr context.
```

Comment:
Should we note the exact file which includes this file? That could be helpful for debugging.

Reply:
I think such a comment is likely to become stale. We could instead explain it in docs. What do you think?

Reply:
I think docs is a better place.